app/styles/components/az-bars.scss

```diff
@@ -0,0 +1,19 @@
+az-bars {
+  width: calc(100% - 96px);
+  div {
+    text-align: right;
+    padding: 5px;
+    height: 30px;
+    span {
+      width: 100%;
+      padding-right: 5px;
+      display: inline-block;
+      &.Creature {background-color: map-get($color-red, '300');}
```

Comment:
No brilliant thoughts here on color substitution. Making them all grey would probably be really bland. We can continue to think about it after merging.

What's the `az-` for?